apps/web/.env

```diff
@@ -1,2 +1,9 @@
 NEXT_PUBLIC_MM_API_URL=https://linked-pool.pancakeswap.com/quote-service
-NEXT_PUBLIC_QUOTING_API=https://swap-quoting.pancakeswap.com/quoting-service
\ No newline at end of file
+NEXT_PUBLIC_QUOTING_API=https://swap-quoting.pancakeswap.com/quoting-service
+MOONPAY_URL=https://buy-sandbox.moonpay.com?apiKey=pk_test_1Ibe44lMglFVL8COOYO7SEKnIBrzrp54
+MOONPAY_TEST_SECRET_KEY=sk_test_7zfPNfcZdStyiktn3lOJxOltGttayhC
+MOONPAY_TEST_API_KEY=sk_test_7zfPNfcZdStyiktn3lOJxOltGttayhC
+REACT_APP_MOONPAY_API="https://api.moonpay.com"
+REACT_APP_MOONPAY_PUBLISHABLE_KEY="pk_test_1Ibe44lMglFVL8COOYO7SEKnIBrzrp54"
+PRIVATE_KEY=-----BEGIN PRIVATE KEY-----\nMIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAMfRrRXD0Jw4+5sOnrwdyUPirVRSw6L15ZNIMCuRA+XNo2+fUVCsbamdJhyn37uNaUswsba99oOzNztOpKcMnu/YnGMowdci5IIp8luPJjdQSTgy1WlHTCCuIenddNUQ9X3RNNlwTxxUYMnzIJe27LQUw92zjaCd0GNVsFxiGLixAgMBAAECgYEAimannSipCFvLogWJWsDC09tgtMQyr1/kKJRakQJ7kaNSbZELEHat6kpnyHdaVxzGbocGwI1o/qWWPuTk1LPALSTh2VS5ePLRxRfXXYgfEcTeJ3/50INlq+A9pRRG1dFQLfxlv5jbu8nWCSIaQCpX6QhUMzduJIrwFl9mlzWcvbUCQQDsAtX+RH/NcT2aYMz9I5fQnZbfxrwoD6k3ipQunjPkqQ63s+e9p+br0mKpDlnCuhIqKE3MVQwp99d28XPdXDKjAkEA2L4hAAqK5TTH+N/6xnpgLV8ksvOxh/DNwu6VbZ9DxjFmyggMGtf0OOhwwHqYqJBvNvpyJ/1+WyEjVL/rgxKwmwJAeMtuyYg3vWfTNtuKr5lzD3RJD1nBXeO+IU5NT5rrRoJ8pOtwlfKEz3hLxOkyaecgevFL+GQjKKvQXPFx8v9IbwJAGzs4FjM85DNPdJtffJRfhRPRHpKzmWrNz1H+gDlkYIBQc82zB6ReUNATant8tqD++B9bPF8DAxwUvk8mnd85RQJAViywe5reNeMQGmKxmovZYfzo6oVveJrTSMG4srQBpw5EmpVS3kdqbS2HhhcAtrY7Aikpj4EG/mQfNAwqvKi8wQ==\n-----END PRIVATE KEY-----
```

Comment:
We better keep this kind of private key local and avoid pushing to remote even though it's for testing purpose. We can have the variable name defined in the `.env.example`

Reply:
all these have been move to my BE rep